src/api/modules/__tests__/dx-mcm.test.ts

```diff
@@ -49,28 +64,52 @@ describe('DX Multi-Channel Message Module', () => {
   describe('markRead', () => {
     it('marks a user message as read', async () => {
       const updatedUserMessage = { ...mockedUserMessages[0], status: 'READ' };
-      nock(DX_MCM_BASE_URL).get(markReadPath(osuId, onid, messageId)).reply(200, {
-        action: 'user-message-mark-read',
-        userMessage: updatedUserMessage,
-      });
+      server.use(
+        rest.get(MARK_READ_PATH_MCM_URL, async (req, res, ctx) => {
+          return res(
+            ctx.status(200),
+            ctx.json({
+              action: 'user-message-mark-read',
+              userMessage: updatedUserMessage,
+            }),
+          );
+        }),
+      );
       mockedGetCache.mockResolvedValue(undefined);
       const result = await markRead(osuId, onid, messageId);
-      expect(result).toMatchObject({ ...mockedUserMessages[0], status: 'READ' });
+      expect(result).toMatchObject({
+        ...mockedUserMessages[0],
+        status: 'READ',
+      });
       expect(result).not.toMatchObject(mockedUserMessages[0]);
     });
 
     it('marks all user messages as read', async () => {
-      nock(DX_MCM_BASE_URL).get(markReadPath(osuId, onid, 'all')).reply(200, {
-        action: 'user-messages-mark-all-read',
-        message: '1 marked as read.',
-      });
+      server.use(
+        rest.get(
+          new RegExp(String.raw`${markReadPath(osuId, onid, 'all')}`),
```

Comment:
didn't use MARK_READ_PATH_MCM_URL here since the call to markReadPath here uses different parameters